cmd/swapcli/main.go

```diff
@@ -772,20 +792,34 @@ func runGetSwapTimeout(ctx *cli.Context) error {
 	return nil
 }
 
-func printOffer(o *types.Offer, index int, indent string) {
+func printOffer(o *types.Offer, index int, indent string) error {
 	if index > 0 {
 		fmt.Printf("%s---\n", indent)
 	}
+
+	xRate := o.ExchangeRate
+	minETH, err := xRate.ToETH(o.MinAmount)
+	if err != nil {
+		return err
+	}
+	maxETH, err := xRate.ToETH(o.MaxAmount)
+	if err != nil {
+		return err
+	}
```

Comment:
It's a little ugly that we have to do error handling here. In theory, we will never receive values, like a zero-valued exchange rate triggering division by zero, that would cause this error handling to be invoked.